### PR TITLE
HWP-208: Db data is persistent

### PR DESCRIPTION
**Pull request by BradyMitch**
# Description

With JWT integration right around the corner, I'd like to push this. I've done tests and the data is persistent. Once JWT integration is fully implemented we can test registering, taking down the containers and rebuilding them, then logging in with that user account data that has stayed persistent.

This PR includes the following proposed change(s):

- database data is persistent

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Refactoring / Documentation
- [ ] Version change

if your change is a breaking change, please add `breaking change` label to this PR

## How Has This Been Tested?

Registering a user account and taking down the docker-compose - user-data stays persistent

## Does the change impact or break the Docker build?

- [x] Yes
- [ ] No

If Yes: Has Docker been updated accordingly?

- [x] Yes
- [ ] No

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have made corresponding changes to the documentation
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: app/client/nginx.conf [hunk 1]
```diff
@@ -1,5 +1,5 @@
 server {
-  listen 8080;
+  listen 80;
```
Comment (akroon3r) [inline app/client/nginx.conf:2]: this needs to be 8080 please.

Comment (akroon3r) [reply app/client/nginx.conf:2]: or better yet, roll back the change on this file.

Comment (BradyMitch) [reply app/client/nginx.conf:2]: Hmm okay that breaks the development side though

Comment (BradyMitch) [reply app/client/nginx.conf]: @akroon3r would it work if we left nginx to listen on port 8080, Dockerfile still exposes to 80, and mapped 80:8080 in docker-compose?

Comment (akroon3r) [reply app/client/nginx.conf]: I don't see why we can't experiment with that.

**Review by ZachBourque — APPROVED** [review]

**Pull request merged**